src/clients/travis-client.js

```diff
@@ -5,8 +5,22 @@ import { getActiveLogger } from '../utils/winston';
 const log = getActiveLogger();
 const tyrAgent = 'Travis/1.0';
 const travisApiUrl = 'https://api.travis-ci.org';
+const travisApiUrlPrivate = 'https://api.travis-ci.com';
 const travisApiAccept = 'application/vnd.travis-ci.2+json';
 
+/**
+ * Gets the proper travis api url depending on if the project should be private or not
+ * @param isPrivate is the project private or not
+ * @returns {string}
+ */
+function getTravisApiUrl(isPrivate) {
+  if (isPrivate) {
```

Comment:
I like it this way: `return (isPrivate) ? travisApiUrlPrivate : travisApiUrlPublic;`
;-)
You don't have to... I guess.